components/cd-article-list/cd-article-list.css

```diff
@@ -0,0 +1,68 @@
+.cd-article-list {
+  margin-top: 4.5rem;
+}
+
+.cd-article-list__title {
+  margin: 0 0 1.5rem 0;
+  color: #1f1f1f;
```

Comment:
This should use existing colour variables as a CSS custom property for consistency https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/sass/cd/_cd-variables.scss#L47
`color: var(--cd-grey--light)`